earth_enterprise/rpms/build.gradle

```diff
@@ -178,6 +209,7 @@ ospackage {
 // Build packages for all platforms by default:
 defaultTasks 'osPackage'
 
+
```

Comment:
General comment: If we're adding adding options like `debug` and `optimize` to `build.gradle`, we should pass them on to all `scons` commands, rather than just for outputting `gee_long_version.txt`.  Otherwise, it may be unexpected for a user to have these options work for only some tasks, and be ignored for others.

It's a small change, but we can create a separate ticket for it, if it would help get tasks completed.